pkg/apis/controller/experiments/v1beta1/experiment_types.go

```diff
@@ -191,9 +191,10 @@ const (
 )
 
 type ParameterSpec struct {
-	Name          string        `json:"name,omitempty"`
-	ParameterType ParameterType `json:"parameterType,omitempty"`
-	FeasibleSpace FeasibleSpace `json:"feasibleSpace,omitempty"`
+	Name             string           `json:"name,omitempty"`
+	ParameterType    ParameterType    `json:"parameterType,omitempty"`
+	DistributionType DistributionType `json:"distributionType,omitempty"`
```

Comment:
Based on our last discussion, do you want to move the `Distribution` under `FeasibleSpace` ?

Reply:
Yes, will update this PR before next sync. Out of station right now.

Reply:
Done